omero/users/whatsnew.txt

```diff
@@ -3,17 +3,21 @@ What's new for OMERO 5.3 for users
 
 Updates and new features for OMERO 5.3 include:
 
-- ROI Folders which allow you to organize your ROIs into a hierarchical tree
-  structure of folders so they can be sorted e.g. by phenotypes or ontologies
+- ROI Folders in OMERO.insight which allow you to organize your ROIs into a
+  hierarchical tree structure of folders so they can be sorted e.g. by
+  phenotypes or ontologies (support for these is still upcoming for OMERO.web)
 - support for lookup tables (LUT) and reverse/inverted intensity rendering
-- annotations can now be added to Wells in the clients and the UI for viewing
-  Screen Plate Well data has been redesigned
-- images can now be projected along T
+- support for histograms
+- Wells can now be annotated in the clients, searched by annotations, and a
+  new script allows you to migrate existing annotations made on Images to
+  Wells
```

Comment:
Hmm - @jburel that's maybe a bit technical/detailed for a "What's new for users" page, since regular client users won't have been able to add annotations to Wells before now.